homeassistant/const.py

```diff
@@ -559,10 +559,13 @@ class Platform(StrEnum):
 VOLUME_CUBIC_FEET: Final = "ft³"
 
 VOLUME_GALLONS: Final = "gal"
-"""US gallon (British gallon is not yet supported)"""
+"""US gallon (Please use VOLUME_IMPERIAL_GALLONS for Imperial gallon)"""
 
 VOLUME_FLUID_OUNCE: Final = "fl. oz."
-"""US fluid ounce (British fluid ounce is not yet supported)"""
+"""US fluid ounce (Please use VOLUME_IMPERIAL_FLUID_OUNCES for Imperial fluid ounce)"""
+
+VOLUME_IMPERIAL_GALLONS: Final = "gal (Imperial)"
+VOLUME_IMPERIAL_FLUID_OUNCES: Final = "fl. oz. (Imperial)"
```

Comment:
This is not a unit, we should not add this here at any point. This will bit our asses for the long future to come.

Reply:
This very much is a unit. In the UK a fluid once is 28.4130625 millilitres (per the law)[https://www.legislation.gov.uk/uksi/1995/1804/schedule/made], while in the US customary system it is approx 29.57 ml (per NIST, who define the customary units)[https://www.nist.gov/pml/owm/approximate-conversions-us-customary-measures-metric].

Unlike pint and gallon differences this one is not talked about a lot, but it really is a thing.

However what is not clear to me is if the UK really use this unit anymore in practice. If they don't then adding it would be absurd.

Reply:
> This very much is a unit. 

It is not. It is a different interpretation of the same unit based on the unit system.
The unit name/description is still the same.

Reply:
Why is it acceptable for temperature units (`degrees` is the unit, `celsius` and `farenheit` is the system) but unacceptable for volume?

Reply:
I stay with my stance that we should not pollute the unit with this.